fearless/Modules/Staking/Model/RewardDestination.swift

```diff
@@ -1,6 +1,35 @@
 import Foundation
+import IrohaCrypto
 
-enum RewardDestination {
+enum RewardDestination<A> {
     case restake
-    case payout(account: AccountItem)
+    case payout(account: A)
+}
+
+extension RewardDestination where A == String {
```

Comment:
consider defining `typealias AccountId = String` or? `typealias AccountAddress = String`

Reply:
Actually this is a good idea. Applied this point where I saw. Let's apply this for other places as we go.